### DISCUSSION TITLE
Hand Gesture Flight Control using Apple Ultra 2

### DISCUSSION THREAD

**Discussion by c1b0rg** [bitcraze/discussions#1698] [Ideas]
Could the Apple Ultra 2’s hand gesture firmware be used to control the Crazyflie 2.1?

**Comment by ToveRumar** [comment]
Hi!

This sounds like a really fun and cool idea! 

It might be a bit tricky to figure out how to filter out any rotational movement that could not be applied to the drone. 
It is a cool idea to think of you controlling your drone with the help or your body. 

Our Post Doc Joe is doing research in this realm, human-drone interaction. Some of his work with the Crazyflie can be found here https://www.bitcraze.io/author/joseph/
